Dockerfile

```diff
@@ -12,5 +12,8 @@ EXPOSE 9200
 ENTRYPOINT java -Xmx2048m \
           -Drestolino.classes=target/classes \
           -Drestolino.packageprefix=com.github.onsdigital.zebedee.api \
+          -javaagent:target/dependency/aws-opentelemetry-agent-1.30.0.jar \
```

Comment:
Should this be aws-opentelemetry-agent-1.31.0.jar for consistency with the babbage version?